Gemfile

```diff
@@ -15,7 +15,5 @@ gemspec
 # Git. Remember to move these dependencies to your gemspec before releasing
 # your gem to rubygems.org.
 
-gem "active_force", github: 'openstax/active_force'
-
 # To use a debugger
-# gem 'byebug', group: [:development, :test]
+gem 'byebug'
```

Comment:
we don't want debugger in production do we?

Reply:
The Gemfile is not included with the gem since it's not listed in the gemspec.